### DISCUSSION TITLE
Tiddlywiki Theme: How to Make New Theme?

### DISCUSSION THREAD

**Discussion by atronoush** [TiddlyWiki/TiddlyWiki5#6365] [Q&A]
For creating a new theme in Tiddlywiki how can I find some docs or instruction?

I searched 
- https://tiddlywiki.com/dev/
- https://tiddlywiki.com/

I can find some community examples here https://tiddlywiki.com/#Community%20Themes but no description!

Is there any relation between Theme and page template or they are separate things?

Where can I start?

**Comment by Alzacon** [comment]
The themes are a type of plugins. https://tiddlywiki.com/#Plugins:Plugins%20%5B%5BPlugin%20Types%5D%5D

I don't know how many ways to create plugins there are... but I know via Node or using the Tinka plugin.

The page template is the basic structure( /layout?) of the basic parts/elements. The theme is how these elements (and others) are displayed (position, appereance, ...). The most of themes are using the default page template, but there are themes that has a custom pagetemplate.

You can look into the basics themes in [tiddlywiki.com](https://tiddlywiki.com/). The theme vanilla is the basic theme and others themes depends of vanilla and it only add a custom CSS or custom assets (for example a font).
